serialmfg/serial_resources/component_instance.py

```diff
@@ -105,14 +106,14 @@ def get(identifier):
         return ComponentInstance(returned_instances[0])
     
     @staticmethod
-    def create(identifier, component_name, part_number_id=None):
+    def create(identifier, component_name, part_number=None):
```

Comment:
Args should be updated in the docstring as well as https://docs.serial.io/python/component-instance#create